### DISCUSSION TITLE
Problem with GitHub Student Email

### DISCUSSION THREAD

**Discussion by nikitasnaik** [community/community#113608] [GitHub Education]
### Hi everyone,

I was trying to sign up for a student GitHub account and it was asking me to use my school email. It said it would send a confirmation email to my school email. But my school email only accepts emails from inside our school domain so I won't be able to receive GitHub's confirmation email. Is there a workaround for this or a way I can use my personal email while still registering through my school? 
Thank you.

**Comment by JasonLuuk** [comment]
Hi @nikitasnaik 

My suggestion is that you should ask your school's IT support staff. Some schools provide two email addresses (corresponding to the same email address), one for external emails, and some schools have IT support staff who can help you whitelist external email accounts so that you can receive them.

**Reply by ShaKk0722** [reply]
Hi @JasonLuuk!
I have a question, if you don't mind, can you help me?
When submitting my student card to register for a GitHub student account, how do I know if I was successful? When I press the "Process my application" button, the page reloads, does that mean it was successful? Can I view the status of my application?

**Reply by samus-aran** [reply]
Hi @ShaKk0722  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

**Comment by samus-aran** [comment]
Hi @nikitasnaik 👋 

Depends on the school. Are you able to share what school you are selecting? 

If you are able to reply with that then also know we've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for.
